### PR TITLE
Docs on the new make:test command

### PR DESCRIPTION
**Pull request by bmitch**
Note on new make:test command from: https://github.com/laravel/framework/commit/79b6bfc4f5bc7ab40649c7df4de30a78771b717b

### CODE REVIEW

File: testing.md [hunk 1]
```diff
@@ -30,18 +30,32 @@ You are free to create other testing environment configurations as necessary. Th
...
+         *
+         * @return void
+         */
+        public function testExample()
         {
             $this->assertTrue(true);
         }
     }
 
+
```
Comment (GrahamCampbell) [inline testing.md]: ?

File: testing.md [hunk 2]
```diff
@@ -30,18 +30,32 @@ You are free to create other testing environment configurations as necessary. Th
...
 
+    use Illuminate\Foundation\Testing\WithoutMiddleware;
+    use Illuminate\Foundation\Testing\DatabaseMigrations;
+    use Illuminate\Foundation\Testing\DatabaseTransactions;
+
     class FooTest extends TestCase
     {
-        public function testSomethingIsTrue()
+        /**
+         * A basic test example.
```
Comment (GrahamCampbell) [inline testing.md]: tests generally don't have phpdoc

File: testing.md [hunk 3]
```diff
@@ -30,18 +30,32 @@ You are free to create other testing environment configurations as necessary. Th
...
-To create a test case, simply create a new test file in the `tests` directory. The test class should extend `TestCase`. You may then define test methods as you normally would using PHPUnit. To run your tests, simply execute the `phpunit` command from your terminal:
+To create a new test case, use the `make:test` Artisan command:
+
+    php artisan make:test FooTest
+
+This command will place a new `FooTest` class within your `tests` directory. You may then define test methods as you normally would using PHPUnit. To run your tests, simply execute the `phpunit` command from your terminal.
 
     <?php
 
+    use Illuminate\Foundation\Testing\WithoutMiddleware;
```
Comment (GrahamCampbell) [inline testing.md]: all these imports are unused

**Comment by bmitch** [comment]
Hmm besides the extra line at the bottom (which I can remove), that code is the result of running:

```php artisan make:test FooTest```

Maybe I'm missing something?

File: testing.md (hunk 3 above)
Comment (taylorotwell) [inline testing.md]: That's what the default test case looks like. They are there for convenience so you can quickly use them if you want.

**Pull request merged**